test/Microsoft.AspNetCore.Antiforgery.Test/Internal/DefaultClaimUidExtractorTest.cs

```diff
@@ -16,16 +17,13 @@ public class DefaultClaimUidExtractorTest
             new DefaultObjectPoolProvider().Create(new AntiforgerySerializationContextPooledObjectPolicy());
 
         [Fact]
-        public void ExtractClaimUid_NullIdentity()
+        public void ExtractClaimUid_NullPrincipal()
         {
             // Arrange
             var extractor = new DefaultClaimUidExtractor(_pool);
 
-            // Act
-            var claimUid = extractor.ExtractClaimUid(null);
-
-            // Assert
-            Assert.Null(claimUid);
+            // Act & Assert
+            Assert.Throws<ArgumentNullException>(() => extractor.ExtractClaimUid(null));
```

Comment:
Doesn't this hit a `Debug.Assert()` failure? Just remove this test if so, maybe even regardless since it's not a real scenario.